### PR TITLE
Projection Pushdown optimizer on by default

### PR DESCRIPTION
**Pull request by apilloud**
Kyle turned this on by default for batch in #17107, now we turn it on for streaming. Users are able to opt out of the change with the experiment, preserving upgrade compatibility.

(See https://github.com/apache/beam/pull/17107#discussion_r829543160)

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by apilloud** [comment]
R: @lukecwik 
cc: @ibzib

**Comment by apilloud** [comment]
Run Java PreCommit

**Comment by apilloud** [comment]
Ping @lukecwik

File: CHANGES.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 ## I/Os
 
 * Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* Projection Pushdown optimizer is now on by default for streaming, matching the behavior of batch pipelines since 2.38.0. If you encounter a bug with the optimizer, please file an issue and disable the optimizer using pipeline option `--experiments=disable_projection_pushdown`.
```
Comment (lukecwik) [inline CHANGES.md:31]: Would it be worthwhile to mention something about upgrade compatibility?

**Review by lukecwik — COMMENTED** [review]
Please update release notes to talk about impact on pipeline update compatibility.

Otherwise LGTM.

File: CHANGES.md (hunk 1 above)
Comment (apilloud) [reply CHANGES.md:31]: Added to the "Breaking Changes" section.

**Comment by apilloud** [comment]
Run Java PreCommit

**Pull request merged**